src/applications/disability-benefits/2346/config/form.js

```diff
@@ -58,7 +59,32 @@ const asyncReturn = (returnValue, error, delay = 300) =>
     }, delay);
   });
 
+const transform = form => {
+  //  Define the transformations
+
+  /**
+   * Snake-cases field names
+   * @param query {Object} an object containing query fields
+   * @returns {Object} query object with updated field names
+   */
+  const rubyifyKeys = query =>
+    Object.keys(query).reduce(
+      (queryParams, key) => ({
+        ...queryParams,
+        [snakeCase(key)]: query[key],
+      }),
+      {},
+    );
+
```

Comment:
We get this for free on the API side - they take camel case and convert to snake - Its not necessary for us to do this.

Reply:
Thanks, just updated.